test/JoeMakerV4.test.ts

```diff
@@ -250,7 +253,7 @@ describe("joeMakerV4", function () {
       await this.zap.zapIn(this.avaxUsdc.address, { value: ethers.utils.parseEther("2") })
       await this.joeAvax.transfer(this.joeMakerV4.address, await this.joeAvax.balanceOf(this.dev.address))
       await this.avaxUsdc.transfer(this.joeMakerV4.address, await this.avaxUsdc.balanceOf(this.dev.address))
-      await this.joeMakerV4.convertMultiple([this.joe.address, this.usdc.address], [this.wavax.address, this.wavax.address])
+      await this.joeMakerV4.convertMultiple([this.joe.address, this.usdc.address], [this.wavax.address, this.wavax.address], "100")
       expect(await this.usdc.balanceOf(this.joeMakerV4.address)).to.equal(0)
       expect(await this.joeAvax.balanceOf(this.joeMakerV4.address)).to.equal(0)
       expect(await this.avaxUsdc.balanceOf(this.joeMakerV4.address)).to.equal(0)
```

Comment:
Again, I think we need better assertions here; the expectation is that Bar receives a certain amount of JOE